video_creation/screenshot_downloader.py

```diff
@@ -75,7 +74,7 @@ def get_screenshots_of_reddit_posts(reddit_object: dict, screenshot_num: int):
         print_substep("Launching Headless Browser...")
 
         browser = p.chromium.launch(
-            headless=True
+            headless=False
```

Comment:
Still need to test this, headless False will cause a lot of confusion for the user